modules/custom/moj_resources/src/CategoryFeaturedContentApiClass.php

```diff
@@ -164,42 +172,22 @@ private function promotedNodes($category, $number, $prison)
       $results->condition('field_moj_top_level_categories', $category);
     };
 
-    $results->range(0, $number);
```

Comment:
Removed because with changed ordering you could theoretically have more than 8 matching tags that push all of this data out so should really get everything for tags and nodes, merge then sort..